src/briefcase/commands/dev.py

```diff
@@ -110,6 +117,37 @@ def install_dev_requirements(self, app: AppConfig, **options):
         else:
             self.console.info("No application requirements.")
 
+    def create_environment(self, app: AppConfig, **options):
```

Comment:
What is this method doing that isn't being handled by the context manager itself?